src/tests/ftest/osa/offline_drain.py

```diff
@@ -107,18 +109,27 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
                     continue
                 output = self.pool.drain(rank, t_string)
                 self.print_and_assert_on_rebuild_failure(output)
+                free_space_after_drain = self.pool.get_total_free_space(refresh=True)
 
                 pver_drain = self.pool.get_version(True)
                 self.log.info("Pool Version after drain %d", pver_drain)
                 # Check pool version incremented after pool drain
                 self.assertTrue(pver_drain > (pver_begin + 1), "Pool Version Error:  After drain")
+                self.assertTrue(initial_free_space > free_space_after_drain,
+                                "Free Space error after drain")
```

Comment:
minor - Recommend a more description error message - here and others
```suggestion
                self.assertTrue(initial_free_space > free_space_after_drain,
                                "Expected Free Space after drain to be less than initial")
```

Reply:
Done.